docs/src/snap/howto/install/install-offline.md

```diff
@@ -140,18 +140,19 @@ In [upstream-images.yaml][upstream-imgs] you will have to
 change the sync target to your private registry mirror.
 
 ```yaml
+# upstream-images.yaml
```

Comment:
let's add an example file from our current images (rocks + synced images). (you can use the `k8s list-images` command to see the full list).

this is because `upstream-images.yaml` is serving a different purpose and could confuse people.